themes/bootstrap3/templates/RecordDriver/DefaultRecord/core.phtml

```diff
@@ -1,5 +1,7 @@
 <?php
   $this->metadata()->generateMetatags($this->driver);
+  $canonicalUrl = $this->recordLinker()->getUrl($this->driver, ['excludeSearchId' => true]);
```

Comment:
Is there a reason you put this here instead of in record/view.phtml? I think using the record/view template will cover more scenarios in a more consistent way, since that is the top-level record display template independent of record drivers (and more equivalent to the authority/record.phtml template that you also adjust in this PR).

Reply:
Thanks, you're right! I just moved the code from core.phtml to view.phtml